<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1745</width>
    <height>1155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="Scroller">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>601</width>
      <height>39</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Browse">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>187</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse Files</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>411</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>187</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Quit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>187</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Capacity_CheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>165</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacity</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Enrollment_CheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>181</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Enrollment</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Waitlist_CheckBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>165</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Waitlist</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Vacancy_CheckBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>165</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Vacancy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Shortfall_CheckBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>165</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Shortfall</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1745</width>
     <height>47</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
